src/components/common/MyPoint/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Button from '@components/common/Button';
+import { Common } from '@styles/globalStyle';
+
+const MyPoint: React.FC = () => {
+  const [selectedAmount, setSelectedAmount] = useState<number | null>(null);
+
+  const handleAmountClick = (amount: number) => {
+    setSelectedAmount(amount);
+  };
+
+  return (
+    <Container>
+      <Title>My Point</Title>
+      <PointBalance>
+        {selectedAmount ? `${selectedAmount.toLocaleString()} P` : '0 P'}
+      </PointBalance>
+      <AmountOptions>
+        {[5000, 10000, 20000, 30000].map((amount) => (
```

Comment:
큰 숫자는 아래처럼 numeric separotr를 활용해보셔도 좋을거 같아요
```suggestion
        {[5_000, 10_000, 20_000, 30_000].map((amount) => (
```